src/test/kotlin/no/nb/mlt/wls/order/controller/OrderControllerTest.kt

```diff
@@ -175,16 +258,31 @@ class OrderControllerTest(
                 .mutateWith(csrf())
                 .mutateWith(mockJwt().jwt { it.subject("axiell") })
                 .delete()
-                .uri("/${duplicateOrderPayload.hostName}/${duplicateOrderPayload.hostOrderId}")
+                .uri("/${dop.hostName}/${dop.hostOrderId}")
```

Comment:
Dunno which method of making URI is better/prefrerred/standard/etc, this or the one on line 177. If it's either or, then we should pick one and stick to it.

Reply:
Using var args to make it more consistent across the codebase